it/repositories/application/GeneralApplicationMongoRepositorySpec.scala

```diff
@@ -469,6 +469,41 @@ class GeneralApplicationMongoRepositorySpec extends MongoRepositorySpec with UUI
     }
   }
 
+  "Update application route" should {
+    "return not found if the application does not match required statuses" in {
```

Comment:
Where is the status that we are creating this application that belongs to the list of required status?